### PR TITLE
Twios 25

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Adds This-week-in-os #25.

### CODE REVIEW

File: src/twios/2023-05-29.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- [ember-fastboot/ember-cli-head]
+  [#105](https://github.com/ember-fastboot/ember-cli-head/pull/105) update
+  ember-auto-import devDependency to v2 ([@mansona])
+- [ember-fastboot/ember-cli-head]
+  [#100](https://github.com/ember-fastboot/ember-cli-head/pull/100) unify CI
+  with the default output of ember addon ([@mansona])
+- [ember-learn/cli-guides]
+  [#287](https://github.com/ember-learn/cli-guides/pull/287) update to v3.28
+  with ember-cli-update and update to latest template ([@mansona])
+- [oscard0m/openapi] [#1](https://github.com/oscard0m/openapi/pull/1) git diff
```
Comment (oscard0m) [inline src/twios/2023-05-29.md]: We can remove this one

File: src/twios/2023-05-29.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+[emberjs/ember.js]: https://github.com/emberjs/ember.js
+[embroider-build/embroider]: https://github.com/embroider-build/embroider
+[empress/guidemaker-default-template]:
+  https://github.com/empress/guidemaker-default-template
+[empress/guidemaker]: https://github.com/empress/guidemaker
+[mainmatter/ember-promise-modals]:
+  https://github.com/mainmatter/ember-promise-modals
+[miguelcobain/ember-paper]: https://github.com/miguelcobain/ember-paper
+[octokit/openapi]: https://github.com/octokit/openapi
+[oscard0m/openapi]: https://github.com/oscard0m/openapi
```
Comment (oscard0m) [inline src/twios/2023-05-29.md]: we can remove this

File: src/twios/2023-05-29.md (hunk 1 above)
Comment (emmasofiah2o) [reply src/twios/2023-05-29.md]: Forever or just for this post?

Comment (oscard0m) [reply src/twios/2023-05-29.md]: For all the upcoming posts

File: config/this-week-in-open-source.config.json [hunk 3]
```diff
@@ -63,7 +63,8 @@
         "BlueCutOfficial/site-books",
         "candunaj/template-precompile-scope",
         "emberjs/core-notes",
-        "candunaj/ember-in-viewport"
+        "candunaj/ember-in-viewport",
+        "oscard0m/openapi"
```
Comment (oscard0m) [inline config/this-week-in-open-source.config.json]:
```suggestion
```

**Review by oscard0m — CHANGES_REQUESTED** [review]

**Comment by emmasofiah2o** [comment]
@oscard0m I have committed your suggestion, but I thought you wanted your repo removed?

**Comment by oscard0m** [comment]
> @oscard0m I have committed your suggestion, but I thought you wanted your repo removed?

Yes. My suggestion removed the repo, right?

**Comment by emmasofiah2o** [comment]
> > @oscard0m I have committed your suggestion, but I thought you wanted your repo removed?
> 
> Yes. My suggestion removed the repo, right?

No, you removed it from the config file where it was under "exclude". Meaning it is now included :D

**Comment by oscard0m** [comment]
> > > @oscard0m I have committed your suggestion, but I thought you wanted your repo removed?
> > 
> > 
> > Yes. My suggestion removed the repo, right?
> 
> No, you removed it from the config file where it was under "exclude". Meaning it is now included :D

Ahhh sorry, I did not notice that was the list of exclusions 🤦🏽. So sorry. Yes, it has to be excluded so keep it in the list of exclusions 🙏🏽

**Comment by emmasofiah2o** [comment]
> > > > @oscard0m I have committed your suggestion, but I thought you wanted your repo removed?
> > > 
> > > 
> > > Yes. My suggestion removed the repo, right?
> > 
> > 
> > No, you removed it from the config file where it was under "exclude". Meaning it is now included :D
> 
> Ahhh sorry, I did not notice that was the list of exclusions 🤦🏽. So sorry. Yes, it has to be excluded so keep it in the list of exclusions 🙏🏽

Oki, I'll change it back later!

**Review by oscard0m — APPROVED** [review]

**Pull request merged**